### PR TITLE
Windows vagrant

### PR DESCRIPTION
**Pull request by romulogoncalves**
Here we fixed the regexp for the loopback.

We also solve the problem with glusterfs trying to rebalance after an unsuccessful run of ansible. The rebalance never works because gv0 is not a distributed volume or only contains a brick. Rebalance does not make sense when we are creating the cluster anyway, only when we add more bricks.
When volume is created in the root partition gluster complains saying that it should be a sub-directory or be forces, I opted for force.

Maybe in the future we should look at this example and build something similar:
https://github.com/cloyne/ansible/blob/master/roles/fileserver/tasks/glusterfs.yml

The rest are little fixes and cosmetic to make things more flexible.

### CODE REVIEW

File: roles/common/tasks/main.yml [hunk 1]
```diff
@@ -4,8 +4,8 @@
   with_items: "{{ trusted_networks }}"
 - name: Set hostname to fqdn
   hostname: name='{{ inventory_hostname }}'
-#- name: Dont bind hostname to loopback
-#  replace: dest=/etc/hosts regexp='^.*{{ inventory_hostname }}.*$' replace='' backup=yes
+- name: Dont bind hostname to loopback
+  replace: dest=/etc/hosts regexp='^(127.0.0.1).*{{ inventory_hostname }}.*$' replace='' backup=yes
```
Comment (sverhoeven) [inline roles/common/tasks/main.yml]: When I create a fresh Vagrant machine with `vagrant init ubuntu/xenial64` and `vagrant up` the /etc/hosts
file contains an `127.0.1.1	ubuntu-xenial.localdomain	ubuntu-xenial` entry.

The regex should also remove that line.

File: README.md [hunk 2]
```diff
@@ -184,21 +184,21 @@ ssh -i emma.key root@emma2.$EMMA_DOMAIN uptime
 Create the `hosts` file see `hosts.template` for template.
 Now use ansible to verify login.
 ```
-ansible all --private-key=emma.key -u root -i hosts -m ping
+ansible all --private-key=emma.key -u root --ssh-extra-args="-o StrictHostKeyChecking=no" -i hosts -m ping
```
Comment (sverhoeven) [inline README.md]: The ansible command is becoming a bit large.
Maybe we start using a [./ansible.cfg](http://docs.ansible.com/ansible/intro_configuration.html) file to put the private-key and ssh-extra-arg.

What do you think?

Comment (romulogoncalves) [inline README.md]: I think it is a good idea. 
However, I need to read on how to do it. The best is to open an issue requesting such enhancement.

Comment (sverhoeven) [inline README.md]: Agree, created #21

File: roles/common/tasks/main.yml (hunk 1 above)
Comment (romulogoncalves) [inline roles/common/tasks/main.yml]: It should now be covered by:
https://github.com/nlesc-sherlock/emma/commit/750aa7018d86acb6fbcf27b2b1db37e7501cb766

File: README.md (hunk 2 above)
Comment (romulogoncalves) [inline README.md]: Cool

Comment (romulogoncalves) [inline README.md]: Solved by:
https://github.com/nlesc-sherlock/emma/commit/67eeb768c6f2624add863c1b25a120ccbec4ab1a

**Pull request merged**